src/hooks/useContract.ts

```diff
@@ -87,14 +87,14 @@ export const useGovernorBravo = () => {
 };
 
 // VRT conversion
-export const useVrtConverter = () => {
+export const useVrtConverterProxy = () => {
   const web3 = useWeb3();
-  return useMemo(() => getVrtConverterContract(web3), [web3]);
+  return useMemo(() => getVrtConverterProxyContract(web3), [web3]);
```

Comment:
I'm curious why the name change? Will be change it back after testing?

Reply:
No, it won't change after testing, we are actually interacting with the proxy contract, I want it to be more precise